pkg/kubernetes/tool.html

```diff
@@ -127,89 +168,121 @@
   </div>
 
   <script id="kubernetes-view-tmpl" type="x-template/mustache">
-    <div class="row">
-      <div class="col-md-4">
-        <h3>Minions</h3>
-        <div class="list-group" id="kubernetes-minions">
-        {{#minions}}
-          <a class="list-group-item" data-id="{{id}}">
-            <h4 class="list-group-item-heading">{{id}}</h4>
-            <p class="list-group-item-text">
-            {{#resources}}
-              {{#capacity}}
-                {{#cpu}}
-                  CPU: {{cpu}}
-                {{/cpu}}
-                {{#memory}}
-                  Memory: {{render_memory}}
-                {{/memory}}
-              {{/capacity}}
-            {{/resources}}
-            </p>
-          </a>
-        {{/minions}}
+<div class="row">
```

Comment:
Please don't reindent everything. In general commits should only touch code that has actually changed. It is nearly impossible to review otherwise.